scripts/build/config.js

```diff
@@ -28,7 +28,10 @@ const parsers = [
   },
   {
     input: "src/language-js/parser-flow.js",
-    strict: false,
+    replace: {
+      // `flow-parser` use this for `globalThis`, can't work in strictMode
+      "(function(){return this}())": '(new Function("return this")())',
```

Comment:
Doesn't create CSP problems?

Reply:
It could be, but we are using this for `parser-postcss.js` too, and nobody complains.

https://github.com/prettier/prettier/blob/master/scripts/build/bundler.js#L241

Reply:
:+1: